pyvista/plotting/tools.py

```diff
@@ -92,18 +94,16 @@ def system_supports_plotting():
 
 def _update_axes_label_color(axes_actor, color=None):
     """Set the axes label color (internal helper)."""
-    if color is None:
-        color = pyvista.global_theme.font.color
-    color = parse_color(color)
+    color = Color(color, default_color=pyvista.global_theme.font.color)
     if isinstance(axes_actor, _vtk.vtkAxesActor):
         prop_x = axes_actor.GetXAxisCaptionActor2D().GetCaptionTextProperty()
         prop_y = axes_actor.GetYAxisCaptionActor2D().GetCaptionTextProperty()
         prop_z = axes_actor.GetZAxisCaptionActor2D().GetCaptionTextProperty()
         for prop in [prop_x, prop_y, prop_z]:
-            prop.SetColor(color[0], color[1], color[2])
+            prop.SetColor(color.f_rgb)
             prop.SetShadow(False)
     elif isinstance(axes_actor, _vtk.vtkAnnotatedCubeActor):
-        axes_actor.GetTextEdgesProperty().SetColor(color)
+        axes_actor.GetTextEdgesProperty().SetColor(color.f_rgb)
```

Comment:
I added a `test_box_axes_label_color` test in `test_plotting.py` to satisfy codecov.